data_store/controllers/load_functions.py

```diff
@@ -325,8 +324,12 @@ def add_project_geospatial_relationship(project_models: list[db.Model]) -> list[
 
     if failing_postcode_prefixes:
         sorted_failing_postcode_prefixes = sorted(list(failing_postcode_prefixes))
-        return abort(
-            500, f"Postcode prefixes not found in geospatial table: {', '.join(sorted_failing_postcode_prefixes)}"
-        )
+        raise MissingGeospatialException(sorted_failing_postcode_prefixes)
 
     return project_models
+
+
+class MissingGeospatialException(Exception):
```

Comment:
Might be better off in an `exceptions.py` module?

Reply:
Moved 👍